website/static/js/projectorganizer.js

```diff
@@ -417,8 +421,8 @@ function _showProjectDetails(event, item, col) {
                 };
             postAction = $osf.postJSON(url, postData);
             postAction.done(function () {
-                treebeard.updateFolder(null, theParentNode);
-                triggerClickOnItem.call(treebeard, item);    
+                treebeard.updateFolder(null, treebeard.find(1));
```

Comment:
It's explained in the description, because if the element has multiple locations (for instance projects put to two different folders that are open) changing name of one has to change the name of the other. Reloading the folder completely is the most efficient way to achieve this I think. Otherwise there will need to be a full lookup. 

Brian's note is important here too. What is the suggestion of the team? I can update the code.